packages/strapi/lib/services/core-store.js

```diff
@@ -105,9 +105,28 @@ const createCoreStore = ({ environment: defaultEnv, db }) => {
       }
     }
 
+    async function del(params = {}) {
+      const { key, environment = defaultEnv, type, name, tag = '' } = Object.assign(
+        {},
+        source,
+        params
+      );
+      const prefix = `${type}${name ? `_${name}` : ''}`;
+      const where = {
+        key: `${prefix}_${key}`,
+        environment,
+        tag,
+      };
+      const data = await db.query('core_store').findOne(where);
+      if (data) {
+        await db.query('core_store').delete({ id: data.id });
+      }
+    }
+
     return {
       get,
       set,
+      del,
```

Comment:
you can call it delete just need to name the function deleteFn & do

```js
return {
delete: deleteFn
}
```

Reply:
Great suggestion! Thanks! About the unit tests, is there a way to do it with the presence of a DB call?